content/documentation/pages/1-installation/1-local/2-docker-customize.md

```diff
@@ -105,117 +189,57 @@ Often while debugging you will need to build new, local `spring-cloud-dataflow-s
+<!--END_TABS-->
 
-<!--TIP-->
+The `skipper` service will wait for a debugger to connect on port `6006`.
 
-**Volume Mounting**
+<!--TIP-->
 
-The explicit volume mounting couples docker-compose to your host’s file system, limiting the portability to other machines and operating systems.
-Unlike `docker`, `docker-compose` does not allow volume mounting from the command line (for example, there is no `-v` parameter).
-Instead, you can define a placeholder environment variable (such as `HOST_APP_FOLDER`) in place of the hardcoded path by using `- ${HOST_APP_FOLDER}:/root/apps` and setting this variable before starting docker-compose.
+Allays use your local machine IP address! Do not use the `localhost` name as it would be wrongly resolved inside the DataFlow or SKipper docker containers.
 
 <!--END_TIP-->
 
-Once you mount the host folder, you can register the app starters (from `/root/apps`), with the Data Flow
-[Shell](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#shell)
-or
-[Dashboard](https://docs.spring.io/spring-cloud-dataflow/docs/current/reference/htmlsingle/#dashboard-apps)
-by using the `file://` URI schema. The following example shows how to do
-so:
-
-```bash
-app register --type source --name my-app --uri file://root/apps/my-app-1.0.0.RELEASE.jar
-```
-
-<!--NOTE-->
-
-**Metadata URIs**
-
-You also need to use `--metadata-uri` if the metadata jar is available in the `/root/apps` folder.
+## Integration Testing
 
-<!--END_NOTE-->
-
-**Maven Local Repository Mounting**
-
-To access the host’s local maven repository from Spring Cloud Data Flow you must mount the host maven local repository to a `dataflow-server` and `skipper-server` volume called `/root/.m2/`. The Maven Local Repository location defaults to `~/.m2` for OSX and Linux and `C:\Documents and Settings\{your-username}\.m2` for Windows.
+The self-documented [DataFlowIT.java](https://github.com/spring-cloud/spring-cloud-dataflow/blob/%github-tag%/spring-cloud-dataflow-server/src/test/java/org/springframework/cloud/dataflow/integration/test/DataFlowIT.java) class demonstrates how to reuse the same docker-compose files to build DataFlow integration and smoke tests.
 
-For MacOS or Linux host machines, this looks like the following listing:
+## Multi-platform support
 
-```
-dataflow-server:
-.........
-  volumes:
-    - ~/.m2:/root/.m2
+The [docker-compose-cf.yml](https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%github-tag%/spring-cloud-dataflow-server/docker-compose-cf.yml) adds a remote `Cloud Foundry` account as a Data Flow runtime platform under the name `cf`. You will need to edit the `docker-compose-cf.yml` to add your CF API URL and access credentials.
```

Comment:
This could use some more explanation. Does this mean you run the local Data Flow and Skipper Servers to deploy to remote platforms? If so does it work for task deployments that require share a DB instance with data flow which is not normally exposed externally to CF?

Reply:
You have a point. I've added a short clarification like this
> This configuration connects your local Skipper server to a remote platform such as Kubernetes or Cloud Foundry and allows you deploy streaming data pipelines on those platforms. As a prerequisite it expects the Apache Kafka (or RabbitMQ) binder provisioned on the remote platform.

> The Task/Batch applications require direct database access. But because the Data Flow server and Database run locally you can start Task application only in your local platform! Also because the Scheduler service is not supported by by the Local Deployer, it is not supported by the docker-compose configuration either.